R/pkg/vignettes/sparkr-vignettes.Rmd

```diff
@@ -27,6 +27,9 @@ library(SparkR)
 
 We use default settings in which it runs in local mode. It auto downloads Spark package in the background if no previous installation is found. For more details about setup, see [Spark Session](#SetupSparkSession).
 
+```{r, include=FALSE}
+SparkR:::sparkCheckInstall()
```

Comment:
this has `include=FALSE` so it will run but the code and output will not be included in the vignettes text